backend/models/User.js

```diff
@@ -0,0 +1,75 @@
+const { Schema, model } = require('mongoose');
+const yup = require('yup');
+
+const phoneRegExp = /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
```

Comment:
It would help to comment on what this regex match is matching against for future devs